collector/collector/report_handler.py

```diff
@@ -217,6 +266,11 @@ def collector_post_handler():
         # must pass args as bytes to uwsgi under Python 3
         process_guilties(klass=classification.encode(), id=str(db_rec.id).encode())
 
+    # Save payload file reference if there's one
+    if BINARY_ATTACHMENT[0] <= int(payload_format_version) <= BINARY_ATTACHMENT[1]:
```

Comment:
Similar question here as above (regarding payload format version value range)

Reply:
This is a bad use of payload version. This should be:
```
if attachment is not None:
        attachment.record_id = db_rec.id
        attachment.save()
```